corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -406,6 +406,7 @@ <h1 class="page-header">
             window.onerror = function(message, file, line, col, error) {
                 $.post('/jserror/', {
                     message: message,
+                    page: window.location.href,
```

Comment:
many times the actual url will be helpful rather than the file where the js error occurred. (been noticing this on the recent emails)